src/mrpro/operators/_FourierOp.py

```diff
@@ -59,14 +59,14 @@ def __init__(
         if isinstance(oversampling, float):
             oversampling = SpatialDimension(oversampling, oversampling, oversampling)
 
-        def get_spatial_dims(spatial_dims, dims):
+        def get_spatial_dims(spatial_dims: SpatialDimension, dims: list[int]):
             return [
                 s
                 for s, i in zip((spatial_dims.z, spatial_dims.y, spatial_dims.x), (-3, -2, -1), strict=True)
                 if i in dims
             ]
 
-        def get_traj(traj, dims):
+        def get_traj(traj: KTrajectory, dims: list[int]):
```

Comment:
Sequence[int], #198

Reply:
I will leave it as it is for the moment to avoid inconsistencies in our code... should be handled in a separate PR adressing #198